lib/rspec/sidekiq/matchers/base.rb

```diff
@@ -196,12 +208,58 @@ def on(queue)
           self
         end
 
+        def once
+          set_expected_count :exactly, 1
+          self
+        end
+
+        def twice
+          set_expected_count :exactly, 2
+          self
+        end
+
+        def thrice
+          set_expected_count :exactly, 3
+          self
+        end
+
+        def exactly(n)
+          set_expected_count :exactly, n
+          self
+        end
+
+        def at_least(n)
+          set_expected_count :at_least, n
+          self
+        end
+
+        def at_most(n)
+          set_expected_count :at_most, n
+          self
+        end
+
+        def times
+          self
+        end
+        alias :time :times
+
+        def set_expected_count(relativity, n)
+          n =
+            case n
+            when Numeric then n
```

Comment:
⚠️  Minor but perhaps this should be `Integer` expressly. `Numeric` will include `Float` objects too. And if we set `n` to a Float for comparison ultimately against the size of an array of matching jobs it'll always fail.